src/Controller/OverviewController.php

```diff
@@ -24,22 +30,38 @@ class OverviewController extends AbstractController
    * 
    * @Route("/add", name="add_info")
    */
-  public function add_info(): Response
+  public function add_info(Request $request): Response
   {
 
-    $entityManager = $this->getDoctrine()->getManager(); // SiAAAmply understanding this as a basic "rule" of symfony;
+    $date = getDate();
+    $entityManager = $this->getDoctrine()->getManager(); // Simply understanding this as a basic "rule" of symfony;
```

Comment:
I swear I fixed that typo 3 times at least.

Reply:
It's correct on my end.

Reply:
What about small comments on the ends of blocks of code. It might not always make sense to have it above or below especially if it will cause gaps in otheriwse clean code.

`//$this->assertSame(++$existing_announcements, count($announcement2));
 $this->assertSame($announcement->getSubject(), $announcement2->getSubject());       // used to $announcement2[0];
 $this->assertSame($announcement->getAuthor(), $announcement2->getAuthor());`